.github/workflows/main.yml

```diff
@@ -65,28 +66,47 @@ jobs:
         ./candi.sh -j 2 --packages="dealii"
         cd ~/dealii-candi/tmp/build/deal.II-* && cat detailed.log && make test
 
-  osx-gcc:
-    name: OSX gcc
+  macos-gcc:
+    name: macos-gcc
     runs-on: [macos-latest]
 
     steps:
     - uses: actions/checkout@v2
     - name: setup
       run: |
-        # force cmake version:
-        pip install cmake==3.18.4
+        # Force cmake version 3.20.5
+        pip install cmake==3.20.5
```

Comment:
we could use homebrew cmake now. Currently it ships v3.21 but we can leave it as it is.

Reply:
Maybe we let it for the moment at 3.20.5. If we update the tester we should update simultaneously update the cmake.package?!